### ISSUE TITLE
Classic LTFH needed for examples is not available

### ISSUE THREAD

**Issue by adrianICG** [EmilMiP/LTFHPlus#21]
Hi,

I was trying to follow the examples in R/Example, but was not able to download classic LTFH to source it. The webpage "https://alkesgroup.broadinstitute.org/UKBB/LTFH/" shows some error, and I could not find any other repository or link to it.

Is it possible for you to include the LTFH source code embedded here or to modify those examples to not rely on the external LTFH?

Thanks

**Comment by EmilMiP** [comment]
Hi,

Truthfully, I had forgotten that the example folder was still there. I remember removing it, but it must have snuck back in somehow. 

The examples in that folder have since been updated, modified, and converted to the examples found in [the pkgdown site for the package](https://emilmip.github.io/LTFHPlus/index.html), so I would recommend that you use those instead.. 

I also briefly checked if I could find the LT-FH implementation, but I was not able to find it either. A long time ago I made an implementation of LT-FH by Hujoel et al, where the sampling was based on the Gibbs sampler used in LT-FH++ instead. [The function ](https://emilmip.github.io/LTFHPlus/reference/estimate_gen_liability_ltfh.html) is called `estimate_gen_liability_ltfh()` and is currently classified as an internal function in the package. Internal functions in an R package can be accessed with `:::`, so accessing this function should simply be `LTFHPlus:::estimate_gen_liability_ltfh`. 

At the moment we do not have plans to make LT-FH a formalized part of the package, but if the original implementation is no longer available, we may do it at some point in the future. This is to say, you are more than welcome to use the implementation we have, but the documentation etc is very bare and provded as-is, but it *should* be a plug-in replacement.
